README.md

```diff
@@ -49,11 +49,48 @@ about. Bricks in the library fall into three categories:
 - :performing_arts: ***Staging bricks*** that format data for downstream tasks, such as ML inference
   and data labeling.
 <br></br>
-## :eight_pointed_black_star: Installation
+## :eight_pointed_black_star: Quick Start
+
+Use the following instructions to get up and running with `unstructured` and test your
+installation.
+
+- Install the Python SDK with `pip install unstructured[local-inference]`
+		- If you do not need to process PDFs or images, you can run `pip install unstructured`
+- Install the following system dependencies if they are not already available on your system.
+  Depending on what document types you're parsing, you may not need all of these.
+    - `libmagic-dev`
+    - `poppler-utils`
+    - `tesseract-ocr`
+    - `libreoffice`
+- Run the following to install NLTK dependencies. `unstructured` will handle this automatically
+  soon.
+  - `python -c "import nltk; nltk.download('punkt')"`
+  - `python -c "import nltk; nltk.download('averaged_perceptron_tagger')"`
+- If you are parsing PDFs, run the following to install the `detectron2` model, which
+  `unstructured` uses for layout detection:
+    - `pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"`
```

Comment:
Not specific to this PR, but this says to install 0.6, while in `unstructured-inference` the readme says to install 0.4, while the `Makefile` installs some specific commit that's not either version. We need to get everything on the same page.

Reply:
Yeah, 🤞 we can just get rid of detectron soon